### ISSUE TITLE
Enhancement: parent::beforeAction in docs

### ISSUE THREAD

**Issue by SOHELAHMED7** [tbreuss/yii2-inertia#3]
In docs (readme.md file), -> "Shared data" section

```php
    public function beforeAction()
    {
        $shared = [
            'user' => $this->getUser(),
            'flash' => $this->getFlashMessages(),
            'errors' => $this->getFormErrors()
        ];
        Yii::$app->get('inertia')->share($shared);
        return true;
    }
```

`parent::beforeAction($action)` statement is  not present. It is nice to have there.

So last line of that method should be 

`return parent::beforeAction($action);`

```php
    public function beforeAction()
    {
        $shared = [
            'user' => $this->getUser(),
            'flash' => $this->getFlashMessages(),
            'errors' => $this->getFormErrors()
        ];
        Yii::$app->get('inertia')->share($shared);
        return parent::beforeAction($action);
    }
```

If you are fine with above proposal, I can also help with Pull Request.

**Comment by tbreuss** [comment]
Good catch! Thank you.

It would be nice to get a pull request.

Do you find some time to do so?

**Comment by tbreuss** [comment]
Fixed with bf327d3783b6fd71e4ecb5a057f414ba40b88526. Thanks for reporting.
